DQMOffline/Trigger/python/Tau3MuMonitor_cfi.py

```diff
@@ -0,0 +1,41 @@
+from math import pi
+
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.tau3muMonitoring_cfi import tau3muMonitoring
+
+hltTau3Mumonitoring = tau3muMonitoring.clone()
+
+# DQM directory
+hltTau3Mumonitoring.FolderName = cms.string('HLT/BPH/Tau3Mu/')
+
+# histogram binning
+hltTau3Mumonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.uint32( 40  ),
+  xmin  = cms.double(- 0.5),
+  xmax  = cms.double( 99.5),
+)
+hltTau3Mumonitoring.histoPSet.etaPSet = cms.PSet(
+  nbins = cms.uint32( 10  ),
+  xmin  = cms.double(- 2.6),
+  xmax  = cms.double(  2.6),
+)
+hltTau3Mumonitoring.histoPSet.phiPSet = cms.PSet(
+  nbins = cms.uint32( 10),
+  xmin  = cms.double(-pi),
+  xmax  = cms.double( pi),
+)
+hltTau3Mumonitoring.histoPSet.massPSet = cms.PSet(
+  nbins = cms.uint32( 40  ),
+  xmin  = cms.double(  0.5),
+  xmax  = cms.double(  3. ),
+)
+
+hltTau3Mumonitoring.taus = cms.InputTag("hltTauPt10MuPts511Mass1p2to2p3Iso", "Taus") # 3-muon candidates
+
+hltTau3Mumonitoring.GenericTriggerEventPSet.andOr          = cms.bool( False ) # https://github.com/cms-sw/cmssw/blob/76d343005c33105be1e01b7b7278c07d753398db/CommonTools/TriggerUtils/src/GenericTriggerEventFlag.cc#L249
```

Comment:
`andOr` set to `False` should ensure that all the four conditions listed here must be satisfied
https://github.com/cms-sw/cmssw/blob/76d343005c33105be1e01b7b7278c07d753398db/CommonTools/TriggerUtils/src/GenericTriggerEventFlag.cc#L251
so I guess the DCS condition comes as a consequence, correct?